org-node/Cargo.toml

```diff
@@ -23,3 +23,8 @@ tracing-subscriber = "0.2"
 ureq = { version = "2.1.1", features = ["json"] }
 ethers = { version = "0.4" }
 rustc-hex = { version = "2.1" }
+tracing-stackdriver = { git = "https://github.com/adaszko/tracing-stackdriver.git", optional = true }
+
+[features]
+default = []
+gcp = ["tracing-stackdriver"]
```

Comment:
Can we name is feature `tracing-stackdriver`? Would be more obvious.

Reply:
You can't name the feature after a package, unfortunately. Or at least it's always failed for me in the past.

Reply:
The name stackdriver is obsolete, as we've discussed in the previous PR.  I'd argue that what's actually more obvious from the operational standpoint is `gcp` because it doesn't delve into internals of a particular log format, which is just an implementation detail.

Reply:
> You can't name the feature after a package, unfortunately. Or at least it's always failed for me in the past.

A feature is defined by default for each optional package, so this wouldn't require an explicit feature under `[features]`

Reply:
> The name stackdriver is obsolete, as we've discussed in the previous PR. I'd argue that from the operational standpoint, `gcp` is actually more obvious because it doesn't delve into internals of a particular log format, which is just an implementation detail.

The point is more that if a feature is essentially activating an optional dependency, then we can just use the name of the dependency instead, as it's already defined as a feature by cargo.

Reply:
> The point is more that if a feature is essentially activating an optional dependency, then we can just use the name of the dependency instead, as it's already defined as a feature by cargo.

While it may make more sense from the implementation standpoint, it's less intelligible to someone deploying client services.  The latter group is hopefully going to be orders of magnitude bigger.

Reply:
Ok sure.